spec/models/commit_monitor_branch_spec.rb

```diff
@@ -71,6 +72,20 @@
     end
   end
 
+  context ".github_pr_uri" do
```

Comment:
it's conventional to use `#method_name` for instance methods and `.method_name` for class methods. but if you decide to change the implementation from the former to the latter then this can stay as is =)

i think i would use `describe` instead of `context` here. though they do the same thing, `context` is usually used for things like conditions or different arguments that affect the outcome.